gradle-conjure/src/main/java/com/palantir/gradle/conjure/ConjureJavaLocalCodegenPlugin.java

```diff
@@ -117,6 +117,14 @@ private static void configureDependencies(Project project, ConjureExtension exte
         if (Dependencies.isJersey(extension.getJava())) {
             project.getDependencies()
                     .add("api", useJakarta ? Dependencies.JAXRS_API_JAKARTA : Dependencies.JAXRS_API_JAVAX);
+            if (Dependencies.isNotNullAuthAndBodyParams(extension.getJava())) {
+                project.getDependencies()
+                        .add(
+                                "implementation",
+                                useJakarta
+                                        ? Dependencies.JAXRS_VALIDATION_API_JAKARTA
+                                        : Dependencies.JAXRS_VALIDATION_API_JAVAX);
+            }
```

Comment:
I don't like this duplicated piece of code, but I'm merely following the existing code. Refactoring it to share the dependencies' setup logic is outside of the scope of this PR.

Similarly, I wanted to add similar tests for the conjure-java-local setup, but it turns out that we don't have a good way to test anything but object generation right, as this is generating from https://github.com/palantir/conjure/blob/master/conjure-api/src/main/conjure/conjure-api.yml, which doesn't have a service to generate. Adding tests for this is also outside of the scope of this PR.